### PR TITLE
Break out of select loop; publish artifacts in script/release

### PR DESCRIPTION
**Pull request by camertron**
Right now, script/release loops because no `break` exists to escape the `select` command loop 😄

### CODE REVIEW

File: script/release [hunk 1]
```diff
@@ -159,6 +164,11 @@ main() {
     add_changed_files $major $minor $patch
     commit $major $minor $patch
     push $major $minor $patch
+
+    # publish artifacts
+    script/publish
```
Comment (manuelpuyol) [inline script/release:169]: I'm not sure if we want to publish artifacts here since we haven't merged the release PR at this point

Comment (camertron) [reply script/release:169]: Hmm yeah I see what you mean. The `push` function prints a message asking you to merge the PR, then waits for you to press a key. Is that enough or should we keep the steps separated?

Comment (manuelpuyol) [reply script/release:169]: I think it is safer to keep them separated

Comment (camertron) [reply script/release:169]: Ok I'm 🆒 with that 😎

**Review by manuelpuyol — APPROVED** [review]

**Pull request merged**